Gui/opensim/nbproject/project.properties

```diff
@@ -2,7 +2,7 @@ app.icon=branding/core/core.jar/org/netbeans/core/startup/frame48.gif
 app.icon.icns=installer_files/OSX/opensim.icns
 app.name=OpenSim
 app.title=OpenSim
-app.version=4.0.Beta
+app.version=OpenSim-4.0-2018-08-20-a5cbd1bd
```

Comment:
Why is `OpenSim-` part of the version? Just for copying the version to GitHub? I don't think that's necessary. I would prefer for the version to not include `OpenSim-`.

Reply:
I went back and forth on this and I wanted the things showing in the AboutPanel to have the OpenSim-4.0 or OpenSim-4.1 prefix. Since all we have is app.version that's where I stuck it. Again if there's a tag it may happen automatically but hard to predict.